Framework/Core/include/Framework/Variant.h

```diff
@@ -28,6 +28,8 @@ namespace o2::framework
 
 // Do NOT insert entries in this enum, only append at the end (before "Empty"). Hyperloop depends on the order.
 enum class VariantType : int { Int = 0,
+                               Int8,
+                               Int16,
```

Comment:
Do NOT insert entries in this enum, only append at the end (before "Empty"). Hyperloop depends on the order.
(as it says 3 lines above) :)

Reply:
Indeed, that was quite silly of me. Fixed.